src/python/grpcio/grpc/_channel.py

```diff
@@ -197,7 +198,16 @@ def _consume_request_iterator(
   event_handler = _event_handler(state, call, None)
 
   def consume_request_iterator():
-    for request in request_iterator:
+    while True:
+      try:
+        request = next(request_iterator)
+      except StopIteration:
+        break
+      except Exception as e:
+        logging.exception("Exception iterating requests")
```

Comment:
Let's keep the exclamation point at the end of this message.

Also: whoa, I'm a hypocrite! ☹️ The other calls to `logging.exception` throughout the codebase do include the exception in the format string. 😦

My apologies for providing poor examples to follow.